### ISSUE TITLE
Careful.

### ISSUE THREAD

**Issue by YTF1shhh** [mtricht/gamesense-essentials#38]
After leaving this on for a long time the pixels are burnt into the OLED screen. This is expected but if you don't want this don't use it for long periods of time. To prevent this you could add a thing where it moved around the OLED sort of like what the Always On Display does on android phones with OLED displays. I know if this is possible.
Picture: https://prnt.sc/184pqpd

**Comment by mtricht** [comment]
Is it just the icon that is burnt into the OLED screen? Or more? @YTF1shhh

**Comment by ACanadianBear** [comment]
The logo would probably burn in over time as it is static. Could a pixel shift be implemented?

**Comment by YTF1shhh** [comment]
@ACanadianBear yes I said that in the first message Android devices that have OLED displays have pixel shift with their Always On Display to prevent pixel burns. @mtricht The icon and where the numbers are for the clock but you can't really tell what the numbers are it's sort of just all the numbers mixed together.

**Comment by Paft** [comment]
So there's no solution to this until pixel shift is implemented?  When I discovered and tried this that was my first concern. 

Even if you disable icons, there are times when some pixels will almost constantly be lit up,  like the colons in the clock for example.

I think this is the only thing stopping me from using it.

**Comment by Enrico9431** [comment]
Reading this, pixel shift would be a nice implementation indeed. Anyway, how long is a "long time" before pixel burn may happen, minutes, hours, days?
